config/config.go

```diff
@@ -620,6 +619,7 @@ func DefaultP2PConfig() *P2PConfig {
 		TestDialFail:                 false,
 		TestFuzz:                     false,
 		TestFuzzConfig:               DefaultFuzzConnConfig(),
+		PersistentPeersMaxDialPeriod: 0 * time.Second,
```

Comment:
Can we move this up (under `MaxNumOutboundPeers`) to have less changes (hence reduced probability of conflicts?)

Reply:
Sure.